crates/astria-telemetry/src/display.rs

```diff
@@ -37,6 +38,7 @@ pub fn hex<T: AsRef<[u8]> + ?Sized>(bytes: &T) -> Hex<'_> {
 /// A newtype wrapper of a byte slice that implements [`std::fmt::Display`].
 ///
 /// To be used in tracing contexts. See the [`self::hex`] utility.
+#[derive(SerializeDisplay)]
```

Comment:
Introducing `serde_with` into the dependency tree because this avoids extra allocations when serializing these.